src/main/scala/temple/generate/kube/ast/OrchestrationType.scala

```diff
@@ -28,7 +26,7 @@ object OrchestrationType {
     secretName: String,
     envVars: Seq[(String, String)],
     dbStorage: DbStorage,
-    dbLifecycleCommand: LifecycleCommand,
+    dbLifecycleCommand: String,
```

Comment:
Is there a reason this isn’t `LifecycleCommand`?

Reply:
There was a weird bug I couldn't work out when it was, something about scala not being able to find the constructor to `Service` that matched the types, changing to a string worked - plus semantically that value _can_ be anything, so making it a string allows flexibility?